setupcontainer.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Not sure I understood the need for this script. Consider removing it from the codebase.